Scripts/script-SendEmailOnSLABreach.yml

```diff
@@ -0,0 +1,68 @@
+commonfields:
+  id: SendEmailOnSLABreach
+  version: -1
+name: SendEmailOnSLABreach
+script: |-
+  """
+  This script is used to send an email about a breached SLA. The script, by default, sends the email to the assignee of the
+  incident. The email is sent using the send-email command, using all enabled integrations that support the command.
+  """
+
+  def get_owner_email():
+      owner_username = demisto.incidents()[0].get("owner")
```

Comment:
What will happened if `owner` is not defined?

Reply:
The script would fail because there would be no one to send the email to. But the script is meant to send the email to the assignee, so what do we want to do if there is no one assigned? I can make it do nothing but then they wouldn't know that there is no one to inform of the SLA breach

Reply:
I should add indicative error for this case

Reply:
I added an indicative error message but keep in mind that because of the design of error handling of scripts that run on SLA breaches, the error won't be shown